src/main/java/ch/bfh/bti7081/model/User.java

```diff
@@ -29,12 +29,15 @@
     @Column(name = PREFIX + "EMAIL", length = 255)
     private String email;
 
-    @Column(name = PREFIX + "PASSWORD", length = 255)
+    @Column(name = PREFIX + "PASSWORD", length = 512)
     private String password;
 
     @Column(name = PREFIX + "PERMISSION")
     private Integer permission;
 
+    @Column(name = PREFIX + "SALT", length = 512)
+    private String salt;
```

Comment:
Wosch ds Salt würklech ir DB spichere?

Reply:
wo seuis süsch sichere?
wede bemänglisch dases ir gliche tabelle wie z passwort gspicheret wird, chanider dä biitrag hie empfähle:
https://security.stackexchange.com/questions/17421/how-to-store-salt